course_catalog/tasks.py

```diff
@@ -91,6 +95,20 @@ def get_ocw_data(
     ).Bucket(name=settings.OCW_CONTENT_BUCKET_NAME)
     ocw_courses = generate_course_prefix_list(raw_data_bucket)
 
+    total_course_count = len(ocw_courses)
+    if match_courses is not None:
```

Comment:
Does it make sense to pass this to `generate_course_prefix_list` and filter there?